lib/npg_pipeline/function/product_delivery_notifier.pm

```diff
@@ -233,6 +256,37 @@ sub _write_message_file {
   return;
 }
 
+sub _message_content_changed {
+  my ($self, $file_path, $new_message) = @_;
+
+  my $old_message = $self->_read_message_file($file_path);
+  $self->debug('Comparing old message ', pp($old_message),
+               ' with new message ',     pp($new_message));
+
+  my $changed = 0;
+  # These metadata are sufficient to detect a LIMS change.
+  foreach my $attr (qw{customer_name sample_id
+                       sample_name sample_supplier_name
+                       file_path file_md5 rpt_list}) {
```

Comment:
Could this list be defined elsewhere as a global "constant", with the specific meaning of "sufficient to detect a LIMS change", for this and potentially other functions to use?